### PR TITLE
TCES-54 TIM-Timeline Entries

### PR DESCRIPTION
**Pull request by 3milyfz**
# Description

Created timeline entries:

Employer Entries
- contact entry
- job entry
- placement entry
- note entry

Job Lead Entries
* placement entry
* note entry

Client Entries
* contact entry
* update entry
* placement entry
* note entry

Completed TCES-54

image

Screenshot 2024-02-02 at 5 37 31 PM

Screenshot 2024-02-02 at 5 37 53 PM

Screenshot 2024-02-02 at 5 53 31 PM

Screenshot 2024-02-02 at 5 53 44 PM

image

image

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# Checklist

Please delete options that are not applicable.

- [X] My code follows the style guidelines of this project (required)
- [X] I have performed a self-review of my code (required)
- [X] I have linted and formatted my code (required)
- [X] New and existing unit tests pass locally with my changes (required)
- [x] My changes generate no new warnings

### CODE REVIEW

File: client/src/components/timeline-entry-components/card-template-component/index.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          <DateText>{new Date(dateAdded).toLocaleDateString()}</DateText>
+        </Header>
+        <Body expanded={isExpanded}>{body}</Body>
+        {isLongBody && (
+          <Button
+            variant="text"
+            onClick={toggleExpansion}
+            sx={{ p: 0, color: "#3568E5", width: "fit-content" }}
+          >
+            {isExpanded ? "READ LESS" : "READ MORE"}
```
Comment (DanielDervishi) [inline client/src/components/timeline-entry-components/card-template-component/index.jsx:49]: The font size on this button is a bit big. Also be sure to bold the font here

**Review by DanielDervishi — COMMENTED** [review]
Great work Emily! Just left one comment but after this it should be good to go

File: client/src/components/timeline-entry-components/card-template-component/index.jsx (hunk 1 above)
Comment (3milyfz) [reply client/src/components/timeline-entry-components/card-template-component/index.jsx:49]: Thanks for adding the mock data for testing! Is this okay? 
image

**Review by DanielDervishi — DISMISSED** [review]
Great Work Emily! Be sure to revert the changes to app.jsx before merging in

**Review by selintasman — DISMISSED** [review]
Great work Emily! Thank you for making the changes :)

**Review by DanielDervishi — APPROVED** [review]

**Review by selintasman — APPROVED** [review]

**Pull request merged**